src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipCommitmentsJobsHandler.cs

```diff
@@ -20,11 +23,31 @@ public async Task Handle(ApprenticeshipCreatedEvent message, IMessageHandlerCont
             }
             else
             {
-                await api.CreateApprentice(message.ToApprenticeshipCreated());
+                // create registration
+                var res = await api.CreateApprentice(message.ToApprenticeshipCreated());
+
```

Comment:
Not all responses will need to send an invitation

Reply:
Changed comment, added 'if necessary'

Reply:
OK should I add logic to suppress invitation calls? if so what should the logic be? - in the APIM

Reply:
if APIM returns null, just exit

Reply:
Added null check